lib/atrium/member.rb

```diff
@@ -1,20 +1,141 @@
 module Atrium
   class Member
+    include ::ActiveAttr::Model
+    include ::ActiveAttr::Attributes
+
+    attribute :aggregated_at
+    attribute :guid
+    attribute :identifier
+    attribute :institution_code
+    attribute :metadata
+    attribute :name
+    attribute :status
+    attribute :successfully_aggregated_at
+    attribute :user_guid
+
     ##
-    # POST /users/:user_guid/members
+    # CLASS METHODS
     #
-    def self.create(params)
-      endpoint = "/users/#{params[:user_guid]}/members"
+    def self.create(user_guid:, institution_code:, credentials:)
+      endpoint = "/users/#{user_guid}/members"
+      body = create_params(institution_code, credentials)
+      member_response = ::Atrium.client.make_request(:post, endpoint, body)
+
+      member_params = member_response["member"]
+      ::Atrium::Member.new(member_params)
+    end
+
+    def self.list(user_guid:)
+      endpoint = "/users/#{user_guid}/members"
+      members_response = ::Atrium.client.make_request(:get, endpoint)
+
+      members = members_response["members"].map do |member|
+        ::Atrium::Member.new(member)
+      end
+    end
+
+    def self.read(user_guid:, member_guid:)
+      endpoint = "/users/#{user_guid}/members/#{member_guid}"
+      member_response = ::Atrium.client.make_request(:get, endpoint)
+
+      member_params = member_response["member"]
+      ::Atrium::Member.new(member_params)
+    end
+
+    ##
+    # INSTANCE METHODS
+    #
+    def accounts
+      endpoint = "users/#{self.user_guid}/members/#{self.guid}/account"
+      accounts_response = ::Atrium.client.make_request(:get, endpoint)
+
+      account = accounts_response["accounts"].map do |account|
```

Comment:
Ah, ya. I probably wasn't clear. I meant that since you're just mapping through and creating an `::Atrium::Account`, you won't need to assign it to `account`. i.e. remove the `account =` in `account = accounts_response["accounts"].map do |account|`